kubectl-plugin/kubectl-waiops

```diff
@@ -428,6 +435,19 @@ then
         getSubscriptionStatus "ibm-zen-operator" "ibm-common-services" 
         getSubscriptionStatus "operand-deployment-lifecycle-manager-app" "ibm-common-services" 
 
+        if [ "${CLUSTER_SCOPE_INSTALL}" == "true" ];
```

Comment:
Could we maintain a var to hold the correct NS based on the CLUSTER_SCOPE_INSTALL check and just use that instead of the if/else blocks?  Might simplify the code?

Reply:
Hi @kvijai82 -- one issue I see with doing this is that currently, the sections are organized by the namespace they are in (for example highlighted above, it's `Subscriptions from ibm-common-services namespace`), which makes the if-else blocks sort of necessary (in the current implementation at least), to ensure components like these with varying namespace possibilities go into a `Subscriptions from openshift-operators namespace` alternate section instead.

**However**, we could simplify the code and avoid the if-else block situation by bundling the output of all subscriptions together with no namespace distinction -- so for example, just have a section called `Subscriptions:` to house all of them. This would allow us to remove the if blocks and also reduce the code complexity a bit. This would also be further organized with the [interns' work to include group headers for these sections](https://github.com/IBM/cp4waiops-samples/pull/87#issuecomment-1184388738). We already do something like this with `OperandRequests`, so I think it would make sense to do the same here -- and possibly with other sections.

Wondering your thoughts on this.

Reply:
Thanks for the explanation.  Let's go with this now and we can refactor as part of the broader refactor changes we have upcoming.

Reply:
Talked with Vijai about this and we decided we can refactor this along with the intern work.